README.md

```diff
@@ -20,6 +28,90 @@ it, simply add the following line to your Podfile:
 pod 'TrustlySDK'
 ```
 
+For development and test reasons, you can install the pod from a branch in github:
+```ruby
+pod 'TrustlySDK', :git => 'https://github.com/TrustlyInc/trustly-ios.git', :branch => '<BRANCH_NAME>'
+```
+</details>
+
+<details open>
+<summary>Swift Package Manager - SPM</summary>
+<br />
+TrustlySDK is available through [Swift Package Manager](https://cocoapods.org). To install
+it, simply click on File -> Add packages -> Search or Enter Package URL, paste the github url about this package `https://github.com/TrustlyInc/trustly-ios.git`:
+
+![Add package url](docs/images/swift_package_manager.png)
+
+For production reason choose the `main` branch, but for test or validation choose the branch described in the [release version table](#versions).
+</details>
+<br />
+
+## Usage
+
```

Comment:
Add a reference to the Developer Docs:

"To quickly get up and running with this SDK see the [iOS Quickstart](https://trustly.readme.io/payments/docs/ios-quickstart) in the Trustly Developer Documentation."